crates/shared/src/arguments.rs

```diff
@@ -217,21 +213,6 @@ pub struct Arguments {
     #[clap(long, env, default_value = "1s", value_parser = humantime::parse_duration)]
     pub pool_cache_delay_between_retries: Duration,
 
-    /// The ParaSwap API base url to use.
-    #[clap(long, env, default_value = super::paraswap_api::DEFAULT_URL)]
-    pub paraswap_api_url: String,
-
-    /// Special partner authentication for Paraswap API (allowing higher rater
-    /// limits)
-    #[clap(long, env)]
-    pub paraswap_partner: Option<String>,
-
-    /// The list of disabled ParaSwap DEXs. By default, the `ParaSwapPool4`
-    /// DEX (representing a private market maker) is disabled as it increases
-    /// price by 1% if built transactions don't actually get executed.
-    #[clap(long, env, default_value = "ParaSwapPool4", use_value_delimiter = true)]
-    pub disabled_paraswap_dexs: Vec<String>,
-
     #[clap(long, env)]
     pub zeroex_url: Option<String>,
```

Comment:
Should this one be removed as well?

Reply:
Looks like there are a lot more configuration arguments (related to 1inch, balancer etc. which shouldn't be needed anymore?)

Reply:
Right, forgot a few arguments. The balancer arguments are still used by the http price estimator (i.e. legacy solver thingy). Those can be removed when we migrate the last price estimators to the colocated setup.